### DISCUSSION TITLE
homebrew

### DISCUSSION THREAD

**Discussion by exodusmulaggusi** [Homebrew/discussions#2096] [Everyday usage]
helo someone please help me.....homebrew will not install on my computer because this is what it tells me in the terminal "You are using macOS 10.11.
==> We (and Apple) do not provide support for this old version.
This installation may not succeed.
After installation, you will encounter build failures with some formulae.
Please create pull requests instead of asking for help on Homebrew\'s GitHub,
Twitter or IRC. You are responsible for resolving any issues you experience
while you are running this old version."

**Comment by alebcay** [comment]
You should [upgrade to the latest version of macOS](https://www.apple.com/macos/how-to-upgrade/) that is supported for your machine. There may also be forks of Homebrew out there that support running on older macOS versions but we are unable to offer support for any problems you may encounter with that approach.

**Comment by exodusmulaggusi** [comment]
my macbookpro is the early 2008 version so i realy cannot go over macOS
10.11..... please and also what command do i use in the terminal to upgrade
to may be the latest version of macOS 10.11 iif there is one assuming?

**Reply by alebcay** [reply]
You can try the instructions given at [this site](https://www.howtogeek.com/347920/how-to-download-macos-updates-using-the-terminal/).

**Comment by exodusmulaggusi** [comment]
thanks alot.... le me try it
